cinder/volume/drivers/nexenta/options.py

```diff
@@ -63,31 +61,53 @@
 ]
 
 NEXENTA_CONNECTION_OPTS = [
+    cfg.StrOpt('nexenta_host',
+               default='',
+               help='IP address of NexentaStor Appliance'),
     cfg.StrOpt('nexenta_rest_address',
                deprecated_for_removal=True,
                deprecated_reason='Rest address should now be set using '
                                  'the common param depending on driver type, '
                                  'san_ip or nas_host',
                default='',
-               help='IP address of NexentaEdge management REST API endpoint'),
-    cfg.StrOpt('nexenta_host',
-               default='',
-               help='IP address of Nexenta SA'),
+               help='IP address of NexentaStor management REST API endpoint'),
     cfg.IntOpt('nexenta_rest_port',
                deprecated_for_removal=True,
                deprecated_reason='Rest address should now be set using '
                                  'the common param san_api_port.',
                default=0,
-               help='HTTP(S) port to connect to Nexenta REST API server. '
-                    'If it is equal zero, 8443 for HTTPS and 8080 for HTTP '
-                    'is used'),
+               help='HTTP(S) port to connect to NexentaStor management '
+                    'REST API server. If it is equal zero, 8443 for '
+                    'HTTPS and 8080 for HTTP is used'),
     cfg.StrOpt('nexenta_rest_protocol',
                default='auto',
                choices=['http', 'https', 'auto'],
-               help='Use http or https for REST connection (default auto)'),
+               help='Use http or https for NexentaStor management '
+                    'REST API connection (default auto)'),
+    cfg.FloatOpt('nexenta_rest_connect_timeout',
```

Comment:
why float not int?

Reply:
According to the urllib3 doc: https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#module-urllib3.util.timeout

total (integer, float, or None) 
connect (integer, float, or None)
read (integer, float, or None)